odml/tools/odmlparser.py

```diff
@@ -13,6 +13,10 @@
 from . import xmlparser
 
 
+# FIX ME: Version should not be hardcoded here. Import from odML module after
+#         fixing the circular imports issue.
+odml_version = '1.3.dev0'
+
```

Comment:
This should be the odml format version which would be 1.1 on master (not the api version)